src/styles/globals.css

```
@@ -262,3 +298,664 @@ Needs to be higher than Intercom chatbox (z-index: 2147483000)
+
+html[data-theme-mode='dark'] .sidebar-menu .htw-rounded-sm .htw-text-gray-800 {
+  color: #f2e4ff !important;
+}
+
+html[data-theme-mode='dark'] .sidebar-menu .htw-rounded-sm .htw-text-gray-500,
+html[data-theme-mode='dark'] .sidebar-menu .htw-rounded-sm .htw-text-xs {
+  color: #c9b5df !important;
+}
+
+html[data-theme-mode='dark'] .sidebar-menu .htw-rounded-sm svg path {
+  fill: currentColor !important;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)